localstack/services/events/events_listener.py

```diff
@@ -138,6 +139,45 @@ def handle_put_rule(data: Dict):
     return True
 
 
+def handle_create_connection(data: Dict, headers: Dict):
+    name = data.get("Name")
+    auth_type = data.get("AuthorizationType")
+    errors = []
+
+    pattern = re.compile("^[\\.\\-_A-Za-z0-9]+$")
```

Comment:
Nit: I'd pull this up a level to avoid "recompiling" it every time. Depending on the actual Python interpreter this might not matter as much but still